std/json.d

```diff
@@ -592,13 +619,13 @@ struct JSONValue
     }
 
     /// Implements the foreach $(D opApply) interface for json arrays.
-    int opApply(int delegate(size_t index, ref JSONValue) dg)
+    int opApply(int delegate(size_t index, ref JSONValue) dg) @system
     {
         enforce!JSONException(type == JSON_TYPE.ARRAY,
                                 "JSONValue is not an array");
         int result;
 
-        foreach(size_t index, ref value; store.array)
+        foreach(size_t index, ref value; array)
```

Comment:
Fixed.

Probably should have left it alone; I was hoping to make opApply @safe-friendly earlier but it didn't work out.